corehq/apps/app_manager/templates/app_manager/case_summary.html

```diff
@@ -145,52 +208,14 @@ <h4 class="panel-title panel-title-nolink">
             <li>
                 <span data-bind="html: $root.moduleFormReference(formId)"></span>
                 <!-- ko: if: conditions -->
-                    <ul class="fa-ul" data-bind="visible: $root.showConditions, foreach: conditions">
-                        <li>
-                            <i class="fa fa-sitemap"></i>
-                            <span data-bind="text: question"></span>
-                            <span data-bind="text: operator"></span>
-                            <span data-bind="text: answer"></span>
-                        </li>
-                    </ul>
+                    <div class="fa-ul" data-bind="visible: $root.showConditions, foreach: conditions">
```

Comment:
Was it intentional to no longer make this a list? It's not a big deal, but a little bit unfortunate to lose the semantic HTML.